assets/js/components/HypothesisLoader.js

```diff
@@ -17,22 +17,26 @@ module.exports = class HypothesisLoader {
     try {
       this.window.hypothesisConfig = HypothesisLoader.assembleHypothesisConfig(this.window);
     } catch (e) {
-      console.error('Could\'t assemble valid Hypothesis config from external data, aborting Hypothesis client loading.');
-      console.error('The error was: ', e);
+      this.window.console.error(e);
+      if (typeof this.window.newrelic === 'object') {
```

Comment:
Considered replacing this with a new utils function called something like `logToRUM` (abstracting out the New Relic specific code into it), but the feature in this PR should ship first. The next time Real User Metric logging is added would be the time to do that, and refactor this to use it.